modules/500-openvpn/template_tests/copy_custom_certificate_test.go

```diff
@@ -71,6 +61,7 @@ var _ = Describe("Module :: openvpn :: helm template :: custom-certificate", fun
 	Context("Default", func() {
 		BeforeEach(func() {
 			f.ValuesSetFromYaml("global", globalValues)
+			f.ValuesSet("global.modulesImages", GetModulesImages())
```

Comment:
don\t need here

Reply:
reverted due to failed test
https://github.com/deckhouse/deckhouse-test-1/actions/runs/3154314187/jobs/5131887278